File: C#/AnaliseDeSentimento/AnaliseDeSentimento/Entidades/Termos.cs
using AnaliseDeSentimento.Arquivo;
using AnaliseDeSentimento.Constantes;
using System.Collections.Generic;

namespace AnaliseDeSentimento.Entidades
{
    public class Termos
    {
        public EnumTermos Tipo { get; init; }
        public List<string> Items { get; private set; }

        public Termos(LerArquivo arquivo, EnumTermos tipoTermos)
        {
            Items = arquivo.Linhas ?? new List<string>();
            Tipo = tipoTermos;
        }
    }
}

File: Python/Entidades/Termos.py
from Constantes.EnumTermos import EnumTermos
from Arquivo.LerArquivo import LerArquivo

class Termos:
  def __init__(self, arquivo: LerArquivo, tipoTermos: EnumTermos):
    self.Items = arquivo.Linhas
    self.Tipo = tipoTermos